packages/react-ui/src/components/molecules/UploadField.js

```diff
@@ -111,14 +114,26 @@ function UploadField(props) {
     setFilesText('');
   };
 
-  const dragPlaceholderText = dragOver ? props.dragPlaceholder : props.placeholder;
+  const setPlaceholder = () => {
+    const multipleSuffix = props.multiple ? '(s)' : '';
+    const placeholder = `Drag and drop your file${multipleSuffix} or click to browse`;
+    const dragPlaceholder = `Drop your file${multipleSuffix} here...`;
+
+    let placeholderText;
+    if (dragOver) {
+      placeholderText = dragPlaceholder;
+    } else {
+      placeholderText = props.placeholder || placeholder;
```

Comment:
Is it on purpose that `props.placeholder` is still used, whereas it has been removed from propTypes and TS? Looks like a mistake.

BTW, this change means we won't be able to redefine placeholder from this component, being stuck with English and these predefined texts. Is it ok?

Reply:
Sorry, my mistake about `placeholder`, I hadn't followed the TypeScript base types. So it's ok to use it here. But I think it should be left in proptypes.

And for the second part of my comment, we accept `dragPlaceholder` can't be redefined, then?

Reply:
Proptypes are independent from TypeScript interface about props, yes. Ideally we would do a similar extend on root's proptypes if desired, but I searched for it and it looks like it's not feasible and not required (https://github.com/mui/material-ui/issues/14790). 

Indeed, and this might be a bit controversial, if we have good props definition, I feel like propTypes are probably less relevant now than they were in the past, assuming most our clients will work with TS. Nice little reading: https://medium.com/smallcase-engineering/proptypes-vs-types-in-typescript-45433990a8f9

So if we have to choose I would invest more effort now in TS than in propTypes. Let's try to be pragmatic with the second.

Reply:
`dragPlaceholder` it's only shown when you are dragging a file over the input, I think we don't need to change that copy and it's different from the regular placeholder (this can be redefined by props). Anyway, I'll confirm with design.

Regarding ENG translation, I've seen in cloud-native that we use `react-intl` lib. But I didn't see the same in C4R, in many places copies are hardcoded. There is something similar in C4R?

Reply:
I've ping design twice with no luck. I would say to leave it as it is (dragPlaceholder) for now and review it in the future if design has a strong opinion on it.

Reply:
Confirmed by design: `dragPlaceholder` is not going to change. So it doesn't need to be accessible by props. The copy is going to be the same in every case.